runners/spark/src/test/java/org/apache/beam/runners/spark/metrics/SparkMetricsPusherTest.java

```diff
@@ -108,7 +111,7 @@ public void processElement(ProcessContext context) {
         counter.inc();
         context.output(context.element());
       } catch (Exception e) {
-        e.printStackTrace();
+        throw new UncheckedIOException((IOException) e);
```

Comment:
Can you just LOG.warn here. I don't think that throwing the Exception is the intended use here.